### ISSUE TITLE
Trying to run a stopped app throws app already exists error

### ISSUE THREAD

**Issue by jsilverio22** [acorn-io/acorn#957]
```
acorn run --name foo ../simple-demo
acorn stop foo
acorn run --name foo ../simple-demo
```

results in 

`Error: app "foo" already exists`

**Comment by danielhass** [comment]
Same here:
```
> acorn -v
acorn version v0.4.1+26487bbc

> acorn run -n awesome-acorn .
> acorn stop awesome-acorn
> acorn ps -A
NAME      IMAGE     HEALTHY   UP-TO-DATE   CREATED   ENDPOINTS   MESSAGE
> acorn run -n awesome-acorn -i .
Error: app "awesome-acorn" already exists
```

**Comment by cjellick** [comment]
fixed and released in v0.4.2
